### PR TITLE
updated PR

### PR DESCRIPTION
**Pull request by SallyMcGrath**
## What does this change?

This updates the PR template for the whole repo, with some helpful comments agreed in our call.

## Checklist

- [x] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [ ] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [ ] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,17 +1,24 @@
 ## What does this change?
 
-<!--delete and update as appropriate-->
+<!-- Add a description of what your PR changes here -->
 
 ### Common Content?
 
+<!-- This PR adds content to the common-content module -->
```
Comment (40thieves) [inline .github/pull_request_template.md:7]:
```suggestion
<!-- Does this PR adds content to the common-content module? -->
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
+
 Block/s
```
Comment (40thieves) [inline .github/pull_request_template.md:9]:
```suggestion
- [ ] Block/s
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,17 +1,24 @@
...
 
 ### Common Theme?
 
+<!-- This PR adds a feature or bugfix to the common-theme module.
+Please reference the ticket you are addressing .-->
+
 Issue number: #issue-number
```
Comment (40thieves) [inline .github/pull_request_template.md:16]:
```suggestion
<!-- Does this PR add a feature or bugfix to the common-theme module? -->

- [ ] Yes

<!--Please reference the ticket you are addressing -->

Issue number: #issue-number
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,17 +1,24 @@
...
 
 ### Org Content?
 
+<!--This PR changes a whole module, a sprint, a page, or a block on a single organisation's module.  -->
```
Comment (40thieves) [inline .github/pull_request_template.md:20]:
```suggestion
<!-- Does this PR changes a whole module, a sprint, a page, or a block on a single organisation's module? -->
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -1,17 +1,24 @@
...
+
 Module | Sprint | Page Type | Block Type
```
Comment (40thieves) [inline .github/pull_request_template.md:22]:
```suggestion
- [ ] Module
- [ ] Sprint
- [ ] Page Type
- [ ] Block Type
```

**Review by 40thieves — COMMENTED** [review]
I left a couple of suggestions, but I think this is a good improvement!

File: .github/pull_request_template.md (hunk 5 above)
Comment (SallyMcGrath) [reply .github/pull_request_template.md:25]: No tbh I want them to write what it's changing, not checklist it, just so I know where to look

Comment (40thieves) [reply .github/pull_request_template.md:25]: Fair 👍🏻

**Review by 40thieves — APPROVED** [review]

**Pull request merged**